### ISSUE TITLE
Match CSS with patternfly standards

### ISSUE THREAD

**Issue by SMahil** [fabric8io/fabric8-planner#1929]
Match CSS with patternfly standards
- Remove Flex by component wise
- Use of grid based layout
- Standardised HTML code as per http://codeguide.co/#html

**Comment by debloper** [comment]
Isn't patternfly going flex in upcoming version?
Is going back to grid based layout really worth/necessary?

**Comment by mindreeper2420** [comment]
@debloper A few things on removing flex from OSIO:

1. PatternFly will not be moving to the structure for quite some time (i.e. no timetable given). Even then, it will require a complete re-write of our system in order to consume it, as the entire structure will be changing. PatternFly 4 is only a release candidate and is still built off of Bootstrap 3's grid system.

2. In addition to PatternFly not moving to a new structure for quite a while, the manner in which flex was dropped into OSIO was incorrect (flex was forced on top of Bootstrap's grid system). This essentially made any consumption of newer versions of PatternFly extremely difficult/impossible (recently attempting to upgrade broke the system). In order to properly replicate the designs into code, the system should have stayed on Bootstrap's grid system - UX designs are created with that particular framework in mind, both for ease of consumption and in order to maintain the standard that these products are built with PatternFly.

3. PatternFly-ng, which is the new Angular2+ repo, is built using PatternFly 3's structure. As such, when OSIO consumes PatternFly-ng, OSIO should not have to fork PatternFly-ng and convert it to flex in order to consume it.

In short, going back to the grid system is necessary for the proper consumption of PatternFly and in order for OSIO to maintain consistency across products.

**Comment by mindreeper2420** [comment]
@SMahil @debloper - PatternFly Roadmap that was just released to the public: https://blog.patternfly.org/patternfly-roadmap-update/. As PatternFly 5 is built off of Bootstrap 4, it will not be set for release until Bootstrap 4 is released as a full version.
